app/assets/javascripts/Components/Result/annotation_panel.jsx

```diff
@@ -10,20 +10,22 @@ export class AnnotationPanel extends React.Component {
       overallComment: props.overallComment,
       unsavedChanges: false
     };
+    this.autoSaveOverallComments = this.autoSaveOverallComments.bind(this);
```

Comment:
Instead of using the "bind in constructor" technique, can we use the "class properties" syntax instead:

https://reactjs.org/docs/faq-functions.html#class-properties-stage-3-proposal

since we're using this syntax in a lot of other react components in MarkUs.